modules/ixBidAdapter.js

```diff
@@ -0,0 +1,252 @@
+import * as utils from 'src/utils';
+import { BANNER } from 'src/mediaTypes';
+import { config } from 'src/config';
+import isArray from 'core-js/library/fn/array/is-array';
+import isInteger from 'core-js/library/fn/number/is-integer';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'ix';
+const BANNER_SECURE_BID_URL = 'https://as-sec.casalemedia.com/cygnus';
+const BANNER_INSECURE_BID_URL = 'http://as.casalemedia.com/cygnus';
+const SUPPORTED_AD_TYPES = [BANNER];
+const ENDPOINT_VERSION = 7.2;
+const CENT_TO_DOLLAR_FACTOR = 100;
+const TIME_TO_LIVE = 60;
+const NET_REVENUE = true;
+const isSecureWeb = utils.getTopWindowLocation().protocol === 'https:';
```

Comment:
This line doesn't work for me - `protocol` does not include the colon. Works without the colon.

Reply:
This has been addressed in #2496